main.go

```diff
@@ -40,6 +40,7 @@ func main() {
 	rootCmd.AddCommand(version.NewCmdVersion())
 
 	rootCmd.PersistentFlags().StringVarP(&configMap, "config-map", "c", "", "Configmap name that needs to be mount")
+	rootCmd.PersistentFlags().StringVarP(&secret, "secret", "s", "", "Secret name that needs to be mount")
```

Comment:
what about multiple secrets?

Reply:
@sadlil , What do you mean? What is the use-case?